recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "3.5.3" %}
+{% set variant = "openblas" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 200
+    skip: true                # [win and not py35]
+    features:
+        - vc14                # [win and py35]
+        - blas_{{ variant }}  # [unix]
+
+# NOTE: This installs the minimum dependencies for Simbody on Windows and OSX,
+# i.e., no visualizer, and all of the optional dependencies on Linux, i.e.
+# include the visualizer. blas is the only required dependency, as it will
+# install openblas which also includes lapack. blas is not needed for Windows
+# because Simbody includes blas/lapack for the Windows builds. FreeGLUT is
+# available via anaconda.org and conda-forge but libXi and libXmu are not, so
+# the user will need those to be install via their Linux package manager.
+requirements:
+    build:
+        - toolchain
+        - cmake
+        - blas 1.1 {{ variant }}  # [not win]
+        - openblas 0.2.18*        # [not win]
+        - freeglut                # [linux]
```

Comment:
I'm getting this failure on CircleCI now:

```
[ 63%] Building CXX object Simbody/tests/CMakeFiles/TestCustomConstraints.dir/TestCustomConstraints.cpp.o
/opt/conda/conda-bld/work/simbody-Simbody-3.5.3/Simbody/Visualizer/simbody-visualizer/simbody-visualizer.cpp:84:25: fatal error: GL/glut.h: No such file or directory
     #include <GL/glut.h>
                         ^
compilation terminated.
make[2]: *** [Simbody/Visualizer/simbody-visualizer/CMakeFiles/simbody-visualizer.dir/simbody-visualizer.cpp.o] Error 1
make[1]: *** [Simbody/Visualizer/simbody-visualizer/CMakeFiles/simbody-visualizer.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 63%] Linking CXX executable ../../TestCustomConstraints
[ 63%] Built target TestCustomConstraints
make: *** [all] Error 2
Command failed: /bin/bash -x -e /opt/conda/conda-bld/work/simbody-Simbody-3.5.3/conda_build.sh
```

Seems the glut header isn't available.